CITATION.cff

```diff
@@ -61,8 +61,8 @@ keywords:
   - moisture recycling
   - atmospheric rivers
 license: Apache-2.0
-version: v3.0.1
-date-released: '2024-04-22'
+version: v3.0.0
+date-released: '2023-07-21'
```

Comment:
```suggestion
date-released: '2023-04-05'
```
3.0.0 was released on 5 April. Although I do see now that we did not update the "date-released" line on the CITATION.cff file. https://github.com/WAM2layers/WAM2layers/commit/623db4075311ff04bd7f99df7dae87360efa397a